src/main/resources/application.properties

```diff
@@ -46,4 +46,7 @@ logging.file.name=LogFile.log
 
 ## Global variables
 max.children.amount=10
-max.images.amount=10
\ No newline at end of file
+max.images.amount=10
+location.init.file.path=src/main/resources/sql/fill-table-location.sql
+test.data.location.init.file.path=src/test/resources/LocationCities.txt
```

Comment:
Why do we need such test properties here?

Reply:
Because of without them ApplicationContext is failed to load

Reply:
I've checked - we don't need it at all.